subliminal/providers/opensubtitles.py

```diff
@@ -66,7 +66,7 @@ def get_matches(self, video, hearing_impaired=False):
             if video.title and sanitized_string_equal(self.series_title, video.title):
                 matches.add('title')
             # guess
-            matches |= guess_matches(video, guess_episode_info(self.movie_release_name + '.mkv'))
+            matches |= guess_matches(video, guessit(self.movie_release_name + '.mkv', {'type': 'episode'}))
```

Comment:
No, it can be dropped.
Le 26 janv. 2016 23:48, "Antoine Bertin" <notifications@github.com> a
écrit :

> In subliminal/providers/opensubtitles.py
> <https://github.com/Diaoul/subliminal/pull/564#discussion_r50914250>:
>
> > @@ -66,7 +66,7 @@ def get_matches(self, video, hearing_impaired=False):
> >              if video.title and sanitized_string_equal(self.series_title, video.title):
> >                  matches.add('title')
> >              # guess
> > -            matches |= guess_matches(video, guess_episode_info(self.movie_release_name + '.mkv'))
> > +            matches |= guess_matches(video, guessit(self.movie_release_name + '.mkv', {'type': 'episode'}))
>
> Do we still need the extension?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Diaoul/subliminal/pull/564/files#r50914250>.
>